examples/templates/bio_sockets.h

```diff
@@ -5,6 +5,10 @@
 #include <openssl/ssl.h>
 #include <openssl/err.h>
 
+#if defined(WIN32)
+#pragma comment(lib, "ws2_32.lib")
```

Comment:
It is either this, or linking this from CMake.
@LiamBindle, any preference?

Reply:
Lets link from CMake.